package.json

```diff
@@ -13,7 +13,7 @@
     "@duneanalytics/client-sdk": "^0.2.1",
     "@fvictorio/newton-raphson-method": "^1.0.5",
     "@radix-ui/react-scroll-area": "^1.0.5",
-    "@swapr/ui": "^0.1.3",
+    "@swapr/ui": "^0.1.41",
```

Comment:
Why is it 0.1.41? 😅 Shouldn't be 0.1.4 ?

Reply:
Nope, it's the newest version.